presto-tests/src/test/java/com/facebook/presto/tests/TestLocalBinarySpilledQueries.java

```diff
@@ -66,4 +67,15 @@ private static LocalQueryRunner createLocalQueryRunner()
 
         return localQueryRunner;
     }
+
+    @Override
+    @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
I agree it would be very useful to get spill to work. But it might be with a later diff. So what I should do for now? always use in-memory hash aggregator when ORDER BY is present?